### DISCUSSION TITLE
Game and Watch games won't change to zoomed mode when pressing the select button

### DISCUSSION THREAD

**Discussion by flagone** [eduardofilo/RG350_adam_image#89] [RetroArch topics]
On other devices, when playing a Game and Watch game the select button will put the game into a zoomed mode where the case is not shown and the game fills the screen.

On the RG280V when I press the select button I just get an error message saying the video resolution is not supported.

**Comment by eduardofilo** [comment]
I've been testing and it depends on the game. Some work fine with the original image settings, but I've found others that really don't.
For example with `Baseball`, with the original settings you can see the complete device, but the screen view shows the resolution error.
In that case changing to the following video settings:
* Video filter: Upscale_256x-320x240
* Scaling > Integer Scale: ON
then the screen view works, but the complete device fails.

I am going to close this issue, because I don't think that there are some settings that works on all the games, given the characteristics of this system. If I find a solution in the future, I will reopen the case.

Thanks for your interest in the issue.
